DSCResources/MSFT_xPSSessionConfiguration/MSFT_xPSSessionConfiguration.psm1

```diff
@@ -60,154 +87,285 @@ function Get-TargetResource
         AccessMode             = $accessMode
         Ensure                 = $ensure
     }
+
+    Write-Verbose ($LocalizedData.GetTargetResourceEndMessage -f $Name)
 }
 
+<#
+    .SYNOPSIS
+        Ensures the specified PSSessionConfiguration is its desired state
```

Comment:
Thanks very much for catching this. I've added the missing word in two lines and updated the PR.